docs/conf.py

```diff
@@ -83,9 +83,9 @@
 project = "geoh5py"
 
 # The short X.Y version.
-version = "0.10.0"
+version = "0.11.0"
 # The full version, including alpha/beta/rc tags.
-release = "0.10.0-alpha.1"
+release = "0.11.0-alpha.1"
```

Comment:
```suggestion
release = "0.10.0-alpha.1"
```
Let's bring it now on next release if we can.